addons/payment_stripe/models/payment_provider.py

```diff
@@ -439,6 +439,16 @@ def _stripe_prepare_proxy_data(self, stripe_payload=None):
 
         return {}
 
+    def _run_provider_onboarding_method(self, **kwargs):
```

Comment:
```suggestion
    def _run_provider_onboarding_method(self, menu_id=False, **kwargs):
```
kwargs used in method should be outside kwargs to avoid kwargs manipulation in method